### ISSUE TITLE
Escaping quotes in PostgreSQL

### ISSUE THREAD

**Issue by vladfaust** [sdogruyol/tren#12]
I have a varchar column title in my table and want to populate it with value `I'm good`.

If to follow README and set escape character to `\\'`, the result will be `I\\''m good`, which will be stored as `I\\'m good`. This is not what I want. What I want after escape is `I''m good`, so the stored value would be `I'm good`, as desired.

What I do now is `firstclass_method(query.gsub("'", "''"))` and `{{!query}}` in .sql, but that might be a security issue.

**Comment by crossonrose** [comment]
How can I help?

**Comment by sdogruyol** [comment]
Good point, actually this might be a good feature to have. Do you have any idea on how to handle this?
